src/App.tsx

```diff
@@ -1,14 +1,80 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { Filter } from './types/enumFilter';
 
 export const App: React.FC = () => {
+  const [user, setUser] = useState<User | null>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todo, setTodo] = useState<Todo | null>(null);
+  const [loader, setLoader] = useState(true);
+  const [flag, setFlag] = useState(Filter.All);
+  const [modalLoad, setModalLoad] = useState(true);
+
+  useEffect(() => {
+    getTodos()
+      .then(fetchedTodos => {
+        setTodos(fetchedTodos);
+      })
+      .finally(() => setLoader(false));
+  }, []);
+
+  const getFilteredByFlag = (filterFlag: Filter) => {
+    switch (filterFlag) {
+      case Filter.All:
+        setFlag(Filter.All);
+        getTodos().then(setTodos);
+        break;
+      case Filter.Active:
+        setFlag(Filter.Active);
+        getTodos()
+          .then(fitchedTodos => fitchedTodos.filter(t => !t.completed))
```

Comment:
There is a typo in the variable name `fitchedTodos`. It should be `fetchedTodos` to maintain consistency and avoid confusion.